src/encoded/tests/test_utils.py

```diff
@@ -0,0 +1,45 @@
+import pytest
+from encoded.util import compute_set_difference_one, find_other_in_pair
+
+
+pytestmark = pytest.mark.working
+
+
+def test_compute_set_difference_one():
+    """ Tests that our set difference utility function behaves correctly under various cases """
+    s1 = {1, 2, 3, 4}
+    s2 = {1, 2, 3}
+    assert compute_set_difference_one(s1, s2) == 4
+    s1 = {1, 2, 3, 4}
+    s2 = {1, 2, 3, 4}
+    with pytest.raises(StopIteration):
+        compute_set_difference_one(s1, s2)
+    with pytest.raises(StopIteration):
+        compute_set_difference_one(s2, s1)
+    s1 = {1, 2, 3, 4, 5, 6}
+    s2 = {1, 2, 3, 4}
+    with pytest.raises(RuntimeError):
+        compute_set_difference_one(s1, s2)
+    with pytest.raises(StopIteration):
+        compute_set_difference_one(s2, s1)
+    s1 = {1, 2}
+    s2 = {1}
+    assert compute_set_difference_one(s1, s2) == 2
+
+
+def test_find_other_in_pair():
+    """ Tests the wrapper for the above function """
+    lst = [1, 2]
+    val = 1
+    assert find_other_in_pair(val, lst) == 2
```

Comment:
I might remove the val assignment since this is so simple and do instead:
```
assert find_other_in_pair(1, lst) == 2
assert find_other_In_pair(2, lst) == 1
with pytest.raises(Exception):
    # Not sure what error the code leads to. From the outside, I'd expect ValueError maybe.
    find_other_in_pair(3, lst)
```